frontend/src/hooks/useDebateWS.ts

```
@@ -0,0 +1,186 @@
+                {
+                  reaction: eventData.payload.reaction,
+                  spectatorHash: eventData.payload.spectatorHash,
+                  timestamp: eventData.payload.timestamp,
+                },
+              ];
+              return newReactions;
+            });
+            break;
+
+          case 'presence':
+            const count = eventData.payload.connected || 0;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)